specification/cognitiveservices/data-plane/ContentSafety/preview/2023-11-30-preview/contentsafety.json

```diff
@@ -0,0 +1,3161 @@
+        }
+      },
+      "required": [
+        "text",
+        "category"
+      ]
+    },
+    "AnnotateTextResult": {
+      "type": "object",
+      "description": "The text annotation response.",
+      "properties": {
+        "label": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The label."
+        },
+        "reasoning": {
+          "type": "string",
+          "description": "The reasoning."
+        }
+      },
+      "required": [
+        "label"
+      ]
+    },
+    "Azure.Core.Foundations.Error": {
+      "type": "object",
+      "description": "The error object.",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "One of a server-defined set of error codes."
+        },
+        "message": {
+          "type": "string",
+          "description": "A human-readable representation of the error."
+        },
+        "target": {
+          "type": "string",
+          "description": "The target of the error."
+        },
+        "details": {
+          "type": "array",
+          "description": "An array of details about specific errors that led to this reported error.",
+          "items": {
+            "$ref": "#/definitions/Azure.Core.Foundations.Error"
+          },
+          "x-ms-identifiers": []
+        },
+        "innererror": {
+          "$ref": "#/definitions/Azure.Core.Foundations.InnerError",
+          "description": "An object containing more specific information than the current object about the error."
+        }
+      },
+      "required": [
+        "code",
+        "message"
+      ]
+    },
+    "Azure.Core.Foundations.ErrorResponse": {
+      "type": "object",
+      "description": "A response containing error details.",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/Azure.Core.Foundations.Error",
+          "description": "The error object."
+        }
+      },
+      "required": [
+        "error"
+      ]
+    },
+    "Azure.Core.Foundations.InnerError": {
+      "type": "object",
+      "description": "An object containing more specific information about the error. As per Microsoft One API guidelines - https://github.com/Microsoft/api-guidelines/blob/vNext/Guidelines.md#7102-error-condition-responses.",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "One of a server-defined set of error codes."
+        },
+        "innererror": {
+          "$ref": "#/definitions/Azure.Core.Foundations.InnerError",
+          "description": "Inner error."
+        }
+      }
+    },
+    "DetectUngroundednessGptxResource": {
+      "type": "object",
+      "description": "Connection information for GPT resource.",
+      "properties": {
+        "azureOpenAIEndpoint": {
+          "type": "string",
+          "description": "Endpoint of Azure OpenAI resource."
```

Comment:
Probably we want to mention how customers should grant access briefly here. Pointing customers to the doc needed.